### PR TITLE
Google adwords conversion

### PR DESCRIPTION
**Pull request by salimhb**
Rough implementation based on code generated by google adwords and here: https://support.google.com/adwords/answer/1722054

P.S.: this is not an implementation of the google adwords api.

### CODE REVIEW

**Comment by jhilden** [comment]
looks mergeable to me

File: lib/rack/tracker.rb [hunk 1]
```diff
@@ -12,6 +12,7 @@
 require "rack/tracker/handler_delegator"
 require "rack/tracker/controller"
 require "rack/tracker/google_analytics/google_analytics"
+require "rack/tracker/google_adwords_conversion/google_adwords_conversion"
```
Comment (bumi) [inline lib/rack/tracker.rb]: ich glaube wir sollten dir require in autoloads umbauen.

Comment (jhilden) [inline lib/rack/tracker.rb]: :+1:  but i think we should do it in a separate PR, because it affects all handlers

Comment (bumi) [inline lib/rack/tracker.rb]: +1
